models/users/dim_user_pii.sql

```diff
@@ -1,17 +1 @@
-select
-    external_user_id,
```

Comment:
The extra fields in user_pii were running CH out of memory trying to create the dictionary on the 300M row instance. They are almost always null right now, and I think if we want to start using that data we need a separate "extended" model to join against later.